validator_client/src/beacon_node_health.rs

```diff
@@ -0,0 +1,420 @@
+use itertools::Itertools;
+use serde::{Deserialize, Serialize};
+use std::cmp::Ordering;
+use std::fmt::{Debug, Display, Formatter};
+use std::str::FromStr;
+use types::Slot;
+
+/// Sync distances between 0 and DEFAULT_SYNC_TOLERANCE are considered `synced`.
+/// Sync distance tiers are determined by the different modifiers.
+///
+/// The default range is the following:
+/// Synced: 0..=8
+/// Small: 9..=16
+/// Medium: 17..=64
+/// Large: 65..
+const DEFAULT_SYNC_TOLERANCE: Slot = Slot::new(8);
+const DEFAULT_SMALL_SYNC_DISTANCE_MODIFIER: Slot = Slot::new(8);
+const DEFAULT_MEDIUM_SYNC_DISTANCE_MODIFIER: Slot = Slot::new(48);
+
+type HealthTier = u8;
+type SyncDistance = Slot;
+
+/// Helpful enum which is used when pattern matching to determine health tier.
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+pub enum SyncDistanceTier {
+    Synced,
+    Small,
+    Medium,
+    Large,
+}
+
+/// Contains the different sync distance tiers which are determined at runtime by the
+/// `sync_tolerance` CLI flag and the `sync_distance_modifier` flags.
```

Comment:
looks like the flags have been replaced with `beacon-nodes-sync-tolerances`

Reply:
Fixed in [453e003](https://github.com/sigp/lighthouse/pull/4393/commits/453e003f926e27c09b5a04be7c18f7f6efd8f99f)